### PR TITLE
expands cr3 export permissions

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/15103

the cr3 export was crashing for editor and read-only user types because they did not have sufficient permissions to access all the lookup tables exported in the cr3 csv, so i added read permissions for the three ``atd__`` lookup tables

## Testing
**URL to test:** 
local because it includes database changes

**Steps to test:**
- apply metadata
- log into vzv using the test editor account
- go to the locations page and select a location
- scroll down to the crash list and click the export button
- try exporting all columns, then try exporting a selection
- repeat the steps above using a read-only account

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [x] Code reviewed
- [x] Product manager approved

### CODE REVIEW

File: atd-vzd/metadata/databases/default/tables/public_atd__coordination_partners_lkp.yaml
Comment (johnclary) [inline atd-vzd/metadata/databases/default/tables/public_atd__coordination_partners_lkp.yaml:1]: 😍 i'm just here to say that i'm jealous of your metadata [v3](https://hasura.io/docs/latest/migrations-metadata-seeds/legacy-configs/upgrade-v3/#what-has-changed)!

Comment (tillyw) [reply atd-vzd/metadata/databases/default/tables/public_atd__coordination_partners_lkp.yaml:1]: ngl it's pretty great!

Comment (mddilley) [reply atd-vzd/metadata/databases/default/tables/public_atd__coordination_partners_lkp.yaml:1]: joining the club to say, yes, this diff really shows how we have the good life! 😄

**Review by mddilley — APPROVED** [review]
I replicated the bug without these changes, and then applied the metadata and tested as all roles. I see the bug gone! Thanks for updating and fixing the export! 🙏 🚢  🚀

**Review by frankhereford — COMMENTED** [review]
Maybe it's me -- i take that back -- I'm sure it's me, but for some reason, I have to make this change in `GridTable.js` to make the crash listing page not WSOD with a `Uncaught TypeError: Cannot read properties of undefined (reading 'atd_txdot_crashes')`😢 

image

I also have to make this change below in `DataTable.js`

image

I am not sure what I'm doing differently than y'all, but I'd be so excited to know. This makes me miss typescript so bad too..

## Anyway ..

I don't think it's related to this PR, but if anyone has any insight into what's up and doubly so if they feel this PR is part of the cause, please speak up!

Beyond having to make my own instance work with thoses changes, this looks great.

**Review by patrickm02L — APPROVED** [review]
Send it 🥏

**Comment by patrickm02L** [comment]
## 1/26
Merging on behalf of Tilly

**Comment by frankhereford** [comment]
This was too early to merge. `master` WSOD's for me now. I'm afraid I'm going to have to request changes and block any release with this code in it until this is resolved.

**Comment by frankhereford** [comment]
Please see above edit - to my comment. The problem is real, but the association to this PR was incorrect.

**Pull request merged**